SE-XXX Typed throws.md

```diff
@@ -412,13 +412,42 @@ Everything that applies to the error type of `Result` also applies to error type
 ### Swift grammar additions/changes
 
 What this proposal is about is giving resilience and type safety to an area of the Swift language that lacks of it, ganing both in safety for the developer not just in type system but in terms of reducing the number of possible path error recovering that the developer might have to face when consuming an API.
-The proposed semantics are pretty simple and additive from which we have today:
+
+We define the proposed semantics represented as how the function would be mangled or just as a pseudocode:

```

Comment:
what means mangled in this context? ^^

Reply:
Same link as the other one, is the transformation from reading Swift code into what the interpreter will work with